docs/en/sql-reference/functions/string-search-functions.md

```diff
@@ -378,6 +378,57 @@ For other regular expressions, the code is the same as for the ‘match’ funct
 
 The same thing as ‘like’, but negative.
 
+## iLike {#ilike}
+
+Case insensitive variant of function `like`. It should respect UTF-8 and work similar to `positionCaseInsensitiveUTF8`.
+
+**Syntax**
+
+``` sql
+ilike(haystack, pattern)
+```
+
+**Parameters**
+
+-   `haystack` — Input string. [String](../../sql-reference/syntax.md#syntax-string-literal).
+-   `pattern` — Regular expression with [re2 syntax](https://github.com/google/re2/wiki/Syntax). Must contain groups, each group enclosed in parentheses. If pattern contains no groups, an exception is thrown. [String](../../sql-reference/syntax.md#syntax-string-literal).
```

Comment:
> The `like` function description contains "The regular expression can contain the meta symbols % and _." Does the function really require re2, or is there custom pattern syntax with "%" and "_" special symbols?
> 
> The function definitely must not contain groups. In fact, I'm not sure that it can contain groups.
> 
> I almost sure, the description of these functions must be similar to the PostgreSQL description https://www.postgresql.org/docs/13/functions-matching.html, and this pattern is not a regular expression at all.

Yes, I agree that this pattern is not a regular expression at all. Can I describe different cases of the pattern in the same as in the PostgreSQL? But that's a lot of words.

Reply:
I didn't see many words about the pattern in the Portgres docs, just a couple of senteces and four examples.